operator/roles/forkliftcontroller/tasks/main.yml

```diff
@@ -80,6 +80,11 @@
       definition: "{{ lookup('template', 'controller/provider-host.yml.j2') }}"
     when: "'kubevirt.io' in api_groups"
 
+  - name: "Setup ovirt-populator deployment"
```

Comment:
we need to align with them.
if they are going to use their own controller - we don't need to deploy here. we might need to add a `when` to know if we wish to deploy or not.
this might be tricky - let's say it's a version that CDI doesn't support vol-pop yet - we are good. but once the user upgrade to CDI that does support - we will somehow need to remove the deployment on our side (or CDI should do it for us) and vice versa.

about CRDs - depend on what they are planning - if it's a generic CRD for a global import usage, we will probably need to change the controller code to support their CRD and drop our CRD. if they are planning to use multiple CRDs (per source), we might tell them to use ours. i don't think it will matter much if we both deploy the same CRD(override?).

Reply:
ok, so how about starting with making the deployment of the controller optional?